learn/on-premise/configure/config-users-auth-providers.md

```diff
@@ -5,11 +5,11 @@ sidebar_label: "Users Onboarding"
 ---
 ---
 
-Configure Users and Authentication Providers. There are two ways to onboard users.
+Configure Users and Authentication Providers.
 
 ## Add Users from Launchpad
 
-- Add from User On-boarding -> Create User Section.
+- Add from Users -> Add New User Section.
 - Authentication will be taken care by Platform.
```

Comment:
Add from Users -> Add New User Section.
Add User to Specific Team from Launchpad Teams section
- Authentication will be taken care by Platform.